### PR TITLE
UI alterations for the System Dry Run

### PR DESCRIPTION
**Pull request by Cybower**

### CODE REVIEW

File: AirSim/Unreal/Environments/Blocks/Content/GPS/GPS_display.uasset
Comment (maplebeans1) [inline AirSim/Unreal/Environments/Blocks/Content/GPS/GPS_display.uasset:1]: This has conflicts w/ the updated GPS coordinates for AP. Do we still need this ?

File: AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Level_Progress_Actor.uasset
Comment (maplebeans1) [inline AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Level_Progress_Actor.uasset:1]: This is modified in your other draft PR. Any conflicts ?

Comment (Cybower) [reply AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Level_Progress_Actor.uasset:1]: Will work these changes into the other PR.

File: AirSim/Unreal/Environments/Blocks/Content/GPS/GPS_display.uasset
Comment (Cybower) [reply AirSim/Unreal/Environments/Blocks/Content/GPS/GPS_display.uasset:1]: Removed changes, I didn't edit anything significant

**Review by maplebeans1 — APPROVED** [review]

**Pull request merged**